tests/scripts/all.sh

```diff
@@ -3621,6 +3619,100 @@ component_test_psa_crypto_config_accel_aead () {
     make test
 }
 
+# The 2 following test components, i.e.
+# - component_test_psa_crypto_config_accel_cipher_aead
+# - component_test_psa_crypto_config_reference_cipher_aead
+# are meant to be used together in analyze_outcomes.py script in order to test
+# driver's coverage for ciphers and AEADs.
+component_test_psa_crypto_config_accel_cipher_aead () {
+    msg "test: crypto config with accelerated cipher and AEAD"
+
+    loc_accel_list="ALG_CBC_NO_PADDING ALG_CBC_PKCS7 ALG_CTR ALG_CFB ALG_OFB ALG_XTS \
+                    ALG_GCM ALG_CCM ALG_CHACHA20_POLY1305 \
+                    KEY_TYPE_DES KEY_TYPE_AES KEY_TYPE_ARIA KEY_TYPE_CHACHA20 KEY_TYPE_CAMELLIA"
+
+    # Configure
+    # ---------
+
+    # Start from the crypto config (no X509 and TLS)
+    helper_libtestdriver1_adjust_config "crypto"
```

Comment:
Question: does something break if we change that to `crypto_full`? If yes, then let's create an issue about it, otherwise (or if it's very easy to fix) let's use `crypto_full`.

(Note: fully agree with the choice of excluding X.509 and TLS for starters, but when given a choice between `xxx` and `xxx_full` we should generally go for the full version if we can.)

Reply:
It just took some guards' fix to make it work (because `full` means `USE_PSA_CRYPTO`), but it was pretty easy ;)